### PR TITLE
fix: "Endpoint timeout lost on reboot of worker hosting NSE and k8s-registry"

### PR DESCRIPTION
**Pull request by denis-tingaikin**
## Motivation

Fixes https://github.com/networkservicemesh/cmd-registry-k8s/issues/376

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -157,10 +163,72 @@ func main() {
...
+
+	nses, err := k8sClient.NetworkservicemeshV1().NetworkServiceEndpoints(cfg.Namespace).List(ctx, v1.ListOptions{})
+
+	if err != nil {
+		logger.Warnf("something went wrong on fetcing nse list: %v", err.Error())
+		return
+	}
+
+	for _, nse := range nses.Items {
+		if nse.Spec.ExpirationTime.AsTime().Local().After(time.Now()) {
```
Comment (zolug) [inline main.go:213]: Before?

**Comment by zolug** [comment]
Hi, Did we just forget about this item PR? It was supposed to be part of the related issue, wasn't it?

**Comment by edwarnicke** [comment]
@denis-tingaikin

**Comment by zolug** [comment]
After trying to recall my memories, I think this might have been the first fix proposal. And the one that got merged covers this part as well. Meaning there should be no need for this PR.

**Comment by denis-tingaikin** [comment]
The problem was fixed by another PR https://github.com/networkservicemesh/sdk-k8s/pull/431.

So cllosing.

**Pull request closed without merging**